src/main/java/ch/njol/unofficialmonumentamod/misc/managers/KeybindingHandler.java

```diff
@@ -0,0 +1,65 @@
+package ch.njol.unofficialmonumentamod.misc.managers;
+
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.util.InputUtil;
+import net.minecraft.text.TranslatableText;
+import net.minecraft.text.Text;
+
+import java.util.ArrayList;
+
+public class KeybindingHandler {
```

Comment:
was linked directly to option thing, seemed like asking users to check 2 places for settings would be too much, but not like I'm gonna keep it anyway

Reply:
Almost all other mods also have keybindings there- I have exactly one mod that has keybindings in its settings, and that is because it offers better customisations with modifier keys.